server/src/test/scala/cromwell/engine/workflow/lifecycle/MaterializeWorkflowDescriptorActorSpec.scala

```diff
@@ -474,6 +482,53 @@ class MaterializeWorkflowDescriptorActorSpec extends CromwellTestKitWordSpec wit
 
       system.stop(materializeWfActor)
     }
+
+    "accept valid memory_retry_multiplier" in {
+      List(validMemoryRetryOptions1, validMemoryRetryOptions2, validMemoryRetryOptions3, validOptions) map { options =>
```

Comment:
TOL: If you move the `List.map` outside of the `""accept valid memory_retry_multiplier" in` you'll be able to have separate test for each value in the list. You'd need to work out a way to give them all individual names but it would allow you to have separate success and failure indications for each separate case that you're testing